spartan/devnet-smoke-tests/main.sh

```diff
@@ -0,0 +1,52 @@
+set -eu -o pipefail
```

Comment:
Have you considered `set -x`? I'm trying to undertand what went wrong in the CI run and it's kinda hard without seeing the commands